.github/workflows/cli-release.yml

```diff
@@ -81,43 +88,33 @@ jobs:
         with:
           credentials_json: "${{ secrets.RILL_BINARY_SA }}"
 
-      - name: Upload install script to CDN bucket
+
+      - name: Release - Upload install script to CDN bucket
         uses: google-github-actions/upload-cloud-storage@v1
+        if: env.PUBLISH_RELEASE == 'true'
         with:
           path: scripts/install.sh
           destination: prod-cdn.rilldata.com/
           headers: |-
             cache-control: no-store
 
-      - name: Create latest version file
-        if: env.PUBLISH_NIGHTLY == 'false'
-        run: |-
-          echo '${{ github.ref_name }}' >> latest.txt
-
-      - name: Upload latest version file to CDN bucket
-        if: env.PUBLISH_NIGHTLY == 'false'
+      - name: Release - Upload latest version file to CDN bucket
+        if: env.PUBLISH_RELEASE == 'true'
         uses: google-github-actions/upload-cloud-storage@v1
         with:
           path: latest.txt
           destination: prod-cdn.rilldata.com/rill/
           headers: |-
             cache-control: no-store
 
-      - name: Publish nightly docker image
-        if: env.PUBLISH_NIGHTLY == 'true'
-        run: |-
-          echo '${{ secrets.DOCKERHUB_PASS }}' | docker login -u '${{ secrets.DOCKERHUB_USER }}' --password-stdin
-          docker tag rilldata/rill:latest rilldata/rill:nightly
-          docker push rilldata/rill:nightly
-
```

Comment:
Should we still continue publishing the non-headless nightly Docker images? They might still be used in the MVP deployments (or just people who prefer it as an install path)

Reply:
they can pull the latest headless image instead of nightly

Reply:
But the headless image won't have UI? So not useful for people trying out on local or trying to self-host the local version